apps/etterlatte-tidshendelser/src/main/kotlin/no/nav/etterlatte/tidshendelser/HendelseDao.kt

```diff
@@ -193,6 +194,36 @@ class HendelseDao(
         }
     }
 
+    fun tilbakestillJobSomIkkeStartetSkikkelig(jobbId: Int) {
+        datasource.transaction {
+            val identifiserteOppgaver =
+                queryOf(
+                    """
+                    SELECT count(*)
+                    FROM hendelse 
+                    WHERE jobb_id = :jobbId
+                    """.trimIndent(),
+                    mapOf("jobbId" to jobbId),
+                ).let { query -> it.run(query.map { it.int(1) }.asSingle) }
+
+            if (identifiserteOppgaver != 0) {
+                throw InternfeilException(
+                    "Kan ikke tilbakestille jobb med id=$jobbId, siden det er $identifiserteOppgaver " +
+                        "som er laget for jobben allerede",
+                )
+            }
+
+            queryOf(
+                """
+                UPDATE jobb 
+                SET status = :statusNy
+                WHERE id = :id
+                """.trimIndent(),
+                mapOf("statusNy" to JobbStatus.NY.name, "id" to jobbId),
+            ).let { query -> it.run(query.asUpdate) }
```

Comment:
Burde man sjekke at denne updaten også returnerer oppdaterte rader for å være sikker?

Reply:
Ja, det er sikkert ikke dumt